templates/50unattended-upgrades.j2

```diff
@@ -1,6 +1,6 @@
 Unattended-Upgrade::Automatic-Reboot "false";
 
 Unattended-Upgrade::Allowed-Origins {
-        "Ubuntu lucid-security";
-//      "Ubuntu lucid-updates";
+        "${distro_id}:${distro_codename}-security";
```

Comment:
Shouldn't this be in the form:

```
"${distro_id} ${distro_codename}-security";
```

(following the examples in Ubuntu's [AutomaticSecurityUpdates](https://help.ubuntu.com/community/AutomaticSecurityUpdates) documentation.)